readme.md

```diff
@@ -0,0 +1,43 @@
+# Bell
+
+Dockerize bell app
+
+## Components
+
+* Bell. It is bell app container build with node.js.
+* Couchdb. It is the database container which is initiated with `db-init`.
+
+## How to use
```

Comment:
I just create https://hub.docker.com/r/treehouses/bell/ and we can actually push the prebuilt image to there.

The readme should contain how to built it manually and how to use the ready-to-go image.